lib/polymer_interop.dart

```diff
@@ -20,98 +20,236 @@ class Polymer {
+  /// one in its parent's children list, or null if the specified element is the
+  /// first one in the list.
+  HtmlElement get previousElementSibling => _proxy['previousElementSibling'];
 
-  Node querySelector(String selector) =>
-      _proxy.callMethod('querySelector', [selector]);
+  /// Read-only property that returns the node immediately preceding the specified
+  /// one in its parent's childNodes list, or null if the specified node is the
+  /// first in that list.
+  Node get previousSibling => _proxy['previousSibling'];
 
-  List<Node> querySelectorAll(String selector) =>
-      _proxy.callMethod('querySelectorAll', [selector]);
+  /// Returns a list of nodes distributed within this. These can be
+  /// dom children or elements distributed to children that are insertion
+  /// points.
+  List<HtmlElement> queryDistributedElements(String selectors) =>
+      _proxy.callMethod('queryDistributedElements', [selectors]);
+
+  /// Returns the first element within the document that matches the specified
+  /// group of selectors.
+  HtmlElement querySelector(String selectors) =>
+      _proxy.callMethod('querySelector', [selectors]);
+
+  /// Returns a non-live NodeList of all elements descended from the element on
+  /// which it is invoked that match the specified group of CSS selectors.
+  List<HtmlElement> querySelectorAll(String selectors) =>
+      _proxy.callMethod('querySelectorAll', [selectors]);
+
+  /// Removes an attribute from the the element.
+  void removeAttribute(String name) =>
+      _proxy.callMethod('removeAttribute', [name]);
+
+  /// Removes the given `node` from the element's `lightChildren`.
+  /// This method also performs dom composition.
+  ///
+  /// The removeChild method is transacted lazily in certain cases for
+  /// performance. In order to interrogate the dom (e.g. offsetHeight,
+  /// getComputedStyle, etc.) immediately after one of these operations, call
+  /// PolymerDom.flush() first.
+  Node removeChild(Node node) => _proxy.callMethod('removeChild', [node]);
+
+  /// Replaces one child node of the specified element with another.
+  Node replaceChild(Node newChild, Node oldChild) =>
+      _proxy.callMethod('replaceChild', [newChild, oldChild]);
+
+  /// Adds a new attribute or changes the value of an existing attribute on the
+  /// element.
+  void setAttribute(String name, String value) =>
+      _proxy.callMethod('setAttribute', [name, value]);
+
+  /// Represents the text content of a node and its descendants.
+  String get text => _proxy['textContent'];
 
-  List<Node> getDistributedNodes() => _proxy.callMethod('getDistributedNodes');
+  /// Sets the text content of a node and its descendants.
+  void set text(String text) {
+    _proxy['textContent'] = text;
+  }
+}
 
-  List<Node> getDestinationInsertionPoints() =>
-      _proxy.callMethod('getDestinationInsertionPoints');
+// TODO: add docs
+class PolymerClassList {
```

Comment:
I think it is maybe not added, because you don't need the DomApi to get the class. But you can just do element.classes (or element.classList for js). But if you set/add a class, then the polymer dom api is needed to make sure "that local DOM and light DOM trees are properly maintained" (or something like that)

Not sure, it could also just be forgotten. I'll ask on their slack channel.